superset/assets/javascripts/explorev2/components/QueryAndSaveBtns.jsx

```diff
@@ -37,6 +37,7 @@ export default function QueryAndSaveBtns(
       className="query"
       onClick={onQuery}
       bsStyle={qryButtonStyle}
+      disabled={errorMessage}
```

Comment:
the `disabled` prop should receive a boolean. 
https://react-bootstrap.github.io/components.html#buttons-disabled